### DISCUSSION TITLE
Doks v1.0.0-alpha.1 🚀

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#1044] [Announcements]
Doks v1.0.0-alpha.1 is finally here! This release restructures Doks as a theme for Hyas (the proper, originally intended way) — more  later

- [Release notes](https://github.com/h-enk/doks/releases/tag/v1.0.0-alpha.1)
- [Live demo](https://v1--doks.netlify.app/)
- [Open in Gitpod](https://gitpod.io/#https://github.com/gethyas/doks-demo)

Scaffold a Doks v1.0.0-alpha.1 project with recommended plugins:

```bash
npm create hyas@latest my-doks-site -- --template doks-recommended
```

**Comment by pensivedog** [comment]
Looks great! And loving the short codes. Excited to see a stable release and setup instructions in Hyas to select this as the starter theme.

One little issue I'm noticing in v1.0.0-alpha1: there's some left-right content shifting when clicking through left-hand navigation. I'm using chrome on Windows 10. Not sure if it's an issue in other browsers/systems.

Please keep up the amazing work on this :)

**Reply by h-enk** [reply]
Thanks, great to hear!

I can't replicate your issue /I'm not sure what it is you're noticing. Do you have more input?

**Reply by pensivedog** [reply]
I'm sorry, I don't have more input about what could be causing it. But here is a video showing what it looks like. However, oddly, I just noticed this isn't happening in an incognito window.


https://user-images.githubusercontent.com/105107450/233313387-b99f694b-570a-4eae-bef2-35849de56640.mp4

**Reply by h-enk** [reply]
Thanks, that helps! Looks like an issue with the vertical height of the sidebar(s) /showing the scrollbar. I will be looking into it.

**Comment by jackeyjack1** [comment]
Thank you for all that you do.

**Comment by coinvf** [comment]
can't integrate showcase from source code?

**Reply by h-enk** [reply]
I will be migrating getdoks.org with the first Doks beta release

**Reply by coinvf** [reply]
I will wait. thank you

**Reply by coinvf** [reply]
Hi!
i wait for your v1 update.

**Comment by vcrlab** [comment]
Can you update v1 getdoks.org?

**Reply by h-enk** [reply]
In due time
